dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/deploy/DefaultApplicationDeployer.java

```diff
@@ -764,15 +764,14 @@ public void preDestroy() {
             }
             onStopping();
 
+            unregisterServiceInstance();
             destroyRegistries();
-            destroyServiceDiscoveries();
```

Comment:
Why remove this line?

Reply:
ServiceDiscoveries have been destroyed in destroyRegistries() and removed from registries